tests/introspection/tests.py

```diff
@@ -217,6 +217,20 @@ def test_get_constraints_indexes_orders(self):
                 indexes_verified += 1
         self.assertEqual(indexes_verified, 3)
 
+    @skipUnlessDBFeature('can_introspect_foreign_keys')
+    def test_handle_space_after_foreign_key(self):
+        """
+        Whitespace between FOREIGN KEY and opening parenthesis should be allowed.
+        """
+        with connection.cursor() as cursor:
+            cursor.fetchone = mock.Mock(return_value=[
+                ('CREATE TABLE table_2 (id INTEGER PRIMARY KEY AUTOINCREMENT,'
+                 ' foreign_key_col INTEGER, FOREIGN KEY (foreign_key_col) REFERENCES'
+                 ' {}(id));').format(Article._meta.db_table)
+            ])
+            relations = connection.introspection.get_relations(cursor, 'table_2')
+        self.assertEquals(relations, {'foreign_key_col': ('id', Article._meta.db_table)})
+
```

Comment:
I've updated the `test_get_relations_alt_format` test case to test multiple create table statements, thanks.